dataland-backend/src/main/kotlin/org/dataland/datalandbackend/entities/BasicCompanyInformation.kt

```diff
@@ -0,0 +1,18 @@
+package org.dataland.datalandbackend.entities
+
+/**
+ * Just the basic data regarding a company stored in dataland
+ */
+interface BasicCompanyInformation {
+    val companyId: String
+
+    val companyName: String
+
+    val headquarters: String
+
+    val countryCode: String
+
+    val sector: String?
+
+    var permId: String?
```

Comment:
What about other identifiers like isin, lei, etc?

Reply:
They are not shown on the /companies page. This only contains data for this page.

Reply:
ok, if in the future other identifiers should be displayed, this would have to be extended.